doc/quickstart/cpp.md

```diff
@@ -58,6 +58,20 @@ cd vcpkg
 
 If the version is out of date, please [create an issue or pull request](https://github.com/Microsoft/vcpkg) on the vcpkg repository.
 
+## Installing mlpack from Conan
+
+The mlpack recipe in [Conan](https://conan.io/) is kept up to date by the Conan team members and the community contributors.
+
+Follow the instructions on [this page on how to set up Conan](https://conan.io/downloads).
+
+Install mlpack:
+
+```shell
+conan install --requires=mlpack/4.3.0 --build=missing
```

Comment:
I don't know conan, but is it possible to use, e.g., `--requires=mlpack` or `--requires=mlpack/latest` or something to get the latest version?  I have some worry about this going out of date or getting overlooked for new mlpack releases.

Reply:
Indeed, changing it. Thanks for the comment!

```suggestion
conan install --requires="mlpack/[*]" --build=missing
```